Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates a new empty dataset in the specified project.
+    /// </para>
+    /// <para type="description">
+    /// Creates a new empty dataset in the specified project from a Dataset Object.  This Dataset can be supplied by object via the 
+    /// pipeline or with the -ByObject parameter, or they can be supplied by value with the flags listed below.  If no Project is 
+    /// specified, the default project will be used.  This cmdlet returns a Dataset object.  Required value parameters are to be 
```

Comment:
The "Required value parameters..." is unnecessary, since `Get-Help <cmdlet name` will provide this same information, along with showing which parameters are optional and which are required.